src/main/java/com/puppycrawl/tools/checkstyle/checks/design/DesignForExtensionCheck.java

```diff
@@ -36,7 +36,7 @@
 
 /**
  * <p>
- * The check finds classes that are designed for extension (subclass creation).
+ * Checks that classes are designed for inheritance.
```

Comment:
I find the word `inheritance` lacking as I believe this checks that methods that could be inherited have a javadoc. This wouldn't come to mine if you said "designed for inheritance". I am thinking coding wise (class is not final, etc...)

Reply:
changed back to `designed for extension (subclass creation)`